packages/odyssey-react-mui/src/EmptyState.tsx

```diff
@@ -76,3 +76,4 @@ const MemoizedEmptyState = memo(EmptyState);
 MemoizedEmptyState.displayName = "EmptyState";
 
 export { MemoizedEmptyState as EmptyState };
+export { MemoizedEmptyState as DataTableEmptyState };
```

Comment:
I'm not sure we want to change the name of this component do we?

Reply:
That would break other folks imports. Also breaks the import in our own `DataTable.tsx` file